templates/problem/vote-ajax.html

```diff
@@ -0,0 +1,42 @@
+<div class="problem-vote-form">
+    <div class="problem-vote-form-header">
+        {% if vote %}
+            <span class="problem-vote-form-title">{{ _('Change vote') }}</span>
+            <span class="problem-vote-date">
+                {%- with vote_date=vote.vote_time|date(_('N j, Y, g:i a')) -%}
+                    {{ _('Last voted on %(date)s', date=vote_date) }}
+                {%- endwith -%}
+            </span>
+        {% else %}
+            <span class="problem-vote-form-title">{{ _('Cast vote') }}</span>
+        {% endif %}
+    </div>
+    <form id="delete-problem-vote-form" action="{{ url('delete_problem_vote', problem.code) }}" method="post">
+        {% csrf_token %}
+    </form>
+    <br>
+    <form id="problem-vote-form" action="{{ url('problem_vote', problem.code) }}" method="post">
+        {% csrf_token %}
+        <span class="problem-vote-form-info">{{ _('Points:') }}</span>
+        <input type="number" step="1" placeholder="{{ problem.points }}" value="{% if vote %}{{ vote.points }}{% endif %}"
```

Comment:
We shouldn't show decimal places on the problem points if there are none. (`floatformat`?)